spec/serializers/assessment_serializer_spec.rb

```diff
@@ -2,14 +2,34 @@
 
 RSpec.describe AssessmentSerializer, type: :serializer do
   subject(:serialized) { described_class.new(assessment:).to_hash }
-  let(:assessment) { build(:assessments_assessment) }
-
-  it 'serializes assessment' do
-    expect(serialized).to eq(
-      {
-        'api_name' => assessment.api_name,
-        'timestamp' => assessment.timestamp,
-      }
-    )
+
+  context 'when it is an onboarding assessment' do
+    let(:assessment) { create(:assessment_log, :onboarding) }
+
+    it 'serializes assessment with incognia_signup_id as id' do
+      expect(serialized).to eq(
+        {
+          'api_name' => assessment.api_name,
+          'id' => assessment.incognia_signup_id,
+          'timestamp' => assessment.created_at,
+        }
+      )
+    end
+  end
+
+  AssessmentLog.api_names.except(:onboarding).values.each do |api_name|
+    context "when it is an #{api_name} assessment" do
+      let(:assessment) { create(:assessment_log, :login) }
```

Comment:
should be like this?

```suggestion
      let(:assessment) { create(:assessment_log, api_name) }
```

Reply:
Yeeeeeees.

Reply:
191c77a44e15d6926facc8cd01f1361facc9e3c1